index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Layout Challenge - DN</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+        <div class="container center-wrapper">
+            <header class="three-row-grid__expand-two">
+                <nav class="left-nav">
+                    <img id="twitter" class="twitter icon-lg" src="/assets/twitter.svg" alt="">
+                    <ul class="nav-list">
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">Home</a></li>
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">Explore</a></li>
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">Notification</a></li>
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">Messages</a></li>
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">Bookmarks</a></li>
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">Lists</a></li>
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">Profile</a></li>
+                        <li><img class="icon-md" src="/assets/generic.svg" alt="generic icon"><a href="#">More</a></li>
+                    </ul>
```

Comment:
As your descriptive class name suggests, this could be improved by being wrapped inside a `<nav>`.